apps/wallet/src/features/Send/InputTransaction/CollectiblesTab.tsx

```diff
@@ -98,7 +98,7 @@ export const CollectiblesTab: FC<Props> = ({ onContinue }) => {
 				placeholder="Recipient account"
 				errorText={recipientInput.message}
 				onChangeText={(receiver) => txActions.update({ receiver })}
-				onFocusOut={handleFocusOutRecipient}
+				onBlur={() => handleFocusOutRecipient(receiver)}
```

Comment:
`handleFocusOutRecipient` could get the value from state, does not need to passing it. So it's just `onBlue={handleFocusOutRecipient}`